product_docs/docs/eprs/7/07_common_operations/11_using_ssl_connections.mdx

```diff
@@ -175,17 +175,21 @@ Before you begin, configure the client for SSL with trigger mode.
 
   In our example, we use the copy of this self-signed certificate and key generated for the database server on the client side.
 
-  The default location of these files is `{user.home}/.postgresql(e.g/var/lib/pgsql/.postgresql/)`. The file location can be overridden using SSL connection parameters or Postgres SSL environmental variables, see [Setting non-default paths using environment variables](setting-non-default-paths-using-environment-variables) for more information.
+  The default location of these files is `{user.home}/.postgresql(e.g/var/lib/edb/.postgresql/)`. The file location can be overridden using SSL connection parameters or Postgres SSL environmental variables, see [Setting non-default paths using environment variables](setting-non-default-paths-using-environment-variables) for more information.
 
+  !!!note
+  If you have installed Replication Server using the RPM install package from the EDB repository, the value of the {user.home} property is `/var/lib/edb/` and you need to create the  `.postgresql` directory in this path to have a valid default client SSL directory path (`/var/lib/edb/.postgresql/`).
+  !!!
+  
   Copy and rename the files:
   ```
-  $ cd /var/lib/pgsql/.postgresql/
+  $ cd /var/lib/edb/.postgresql/
 
-  $ cp /var/lib/pgsql/11/data/server.crt postgresql.crt
+  $ cp /var/lib/edb/11/data/server.crt postgresql.crt
```

Comment:
Please update the path to latest version, it should be 
cp /var/lib/edb/14/data/server.crt postgresql.crt